server/tests/productRoute.spec.js

```diff
@@ -0,0 +1,55 @@
+const request = require("supertest");
+const app = require("../server");
+const { Product, db } = require("../db/index");
+
+beforeAll(() => db.sync());
+
+/*-----------products tests----------------*/
+
+describe("/GET /api/products", () => {
+  it("serves up all products", async done => {
+    const response = await request(app).get("/api/products");
+    expect(response.status).toEqual(200);
+    expect(response.body.length).toBeGreaterThan(1);
```

Comment:
More specificity... Check that the returned thing is either `product like` or that it is specific products.